src/Ops.Web/Areas/Admin/Views/Locations/_AddFeaturesPartial.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@model Ocuda.Ops.Controllers.Areas.Admin.ViewModels.Location.FeatureListViewModel;
+
+@if (Model.Features.Count() == 0)
+{
+    <div class="alert alert-warning">No Items found</div>
+}
+else
+{
+    <div>
+        <table class="table table-sm table-bordered table-striped">
+            <thead>
+                <tr>
+                    <td>Name</td>
+                    <td>Stub</td>
+                    <td>General Body Text</td>
+                    <td width="44px">&nbsp;</td>
```

Comment:
Putting a pixel width in the CSS here is bad. Use a class if necessary, but why are we sizing this at all? If we need to size it, can we size it in `rem` rather than `px` (but only in a class).